app/adapters/config.js

```diff
@@ -4,10 +4,10 @@ import Ember from 'ember';
 import PouchAdapterUtils from 'hospitalrun/mixins/pouch-adapter-utils';
 
 export default Adapter.extend(PouchAdapterUtils, {
-    databaseName: 'config',    
-    db:  Ember.computed.alias('pouchDBService.configDB'),
-    pouchDBService: Ember.inject.service('pouchdb'),
-    
+    databaseName: 'config',
+    database: Ember.inject.service(),
+    db:  Ember.computed.alias('database.configDB'),
```

Comment:
grr... I'm sorry, I'm multi tasking so I'm making mistakes. This Adapter is in-fact extending the PouchAdapter, you can see this at the top `import { Adapter } from 'ember-pouch';` so it needs the `db` property. I'll add it back in.